charts/ingress-nginx/Chart.yaml

```diff
@@ -1,14 +1,13 @@
 annotations:
-  artifacthub.io/changes: |2
+  artifacthub.io/changes: |
      - "ci(helm): fix Helm Chart release action 422 error (#10237)"
      - "helm: Use .Release.Namespace as default for ServiceMonitor namespace (#10249)"
      - "[helm] configure allow to configure hostAliases (#10180)"
      - "[helm] pass service annotations through helm tpl engine (#10084)"
-     - "Update Ingress-Nginx version controller-v1.9.0-beta.0"
-  artifacthub.io/license: Apache-2.0
```

Comment:
This was newly introduced in a recent PR, can we update the template and keep it?

Reply:
ouch the difference between your comment and my hold cancel....